tools/clang/lib/SPIRV/DebugTypeVisitor.cpp

```diff
@@ -99,6 +99,18 @@ DebugTypeVisitor::lowerToDebugType(const SpirvType *spirvType) {
                                              vecType->getElementCount());
     break;
   }
+  case SpirvType::TK_Matrix: {
+    // TODO: I temporarily use a DebugTypeArray for a matrix type.
```

Comment:
There are two points that come to mind:

1) In terms of memory layout, representing a matrix as an array of vectors is probably more realistic. (e.g. float3x4 being an array of 3 float4s).

2) Regardless of whether we use a 2D array or an array of vectors to represent a matrix, majorness is something that should be considered. For example if we're using a 2D array, should it be arr[numRows][numCols] or arr[numCols][numRows]?

These are things that I don't want us to spend time on right now. I'll open an issue so we don't  forget about it.